File: Backend/src/main/java/com/tarikturkdil/dto/UserResponse.java
package com.tarikturkdil.dto;

import lombok.Data;


//Herkesin görebileceği profil (başka kullanıcının profiline bakarken)
@Data
public class UserResponse extends DtoBase{

	private String username;
	
	private String avatarUrl;
	
    private String bio;
}

File: Frontend/app/src/main/java/com/tarikturkdil/photoProject/data/remote/dto/UserResponse.kt
package com.tarikturkdil.photoProject.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class UserResponse(
    val id: Long? = null,
    val username: String,
    val avatarUrl: String? = null,
    val bio: String? = null,
    val createTime: String? = null
)